### DISCUSSION TITLE
Can we nuke editor integration? (YES we are)

### DISCUSSION THREAD

**Discussion by iainctduncan** [iainctduncan/scheme-for-max#111] [Ideas]
I believe the built in editor (that you get on double clicking) is causing more problems than solutions. My feeling is no new user ought to be using it anyway, as it will make them hate the language. It causes issues with the threading support because it is not supposed to run in the high thread. Max's ability to work with other editors seems sketchy, and it's a mess on windows.

This thread is to collect votes against this proposal before I do it... or comments in favour.

**Comment by jonnay** [comment]
Yup. I think we can spend more time on proper editor integration through udp/tcp.

**Comment by iainctduncan** [comment]
OK this is happening now.

**Comment by iainctduncan** [comment]
Gone now, dbl click now shows full path to main source file
